pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-finance-storage</artifactId>
-  <version>8.7.4-SNAPSHOT</version>
+  <version>8.7.5-SNAPSHOT</version>
```

Comment:
no need to change this version manually,
This command will create tag and create new commit with new 8.7.5-SNAPSHOT version automatically

mvn -DautoVersionSubmodules=true release:clean release:prepare -Darguments="-Dmaven.test.skip=true -DskipTests"